README.md

```diff
@@ -34,11 +34,15 @@
 
 ## Requirements
 
-Starting from version `0.23.2` of the library your project is required to use:
+Starting from version `0.23.3` of the library your project is required to use:
```

Comment:
I would consider changing the way we apply the plugin a major change, thus it might make sense to bump up the version to `0.24.0.`. @mvicsokolova WDYT?

Reply:
Right, it'd be better to release `0.24.0`